shardingsphere-transaction/shardingsphere-transaction-core/src/main/java/org/apache/shardingsphere/transaction/context/impl/StandardTransactionContexts.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.transaction.context.impl;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.apache.shardingsphere.infra.database.DefaultSchema;
+import org.apache.shardingsphere.transaction.ShardingTransactionManagerEngine;
+import org.apache.shardingsphere.transaction.context.TransactionContexts;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Standard transaction contexts.
+ */
+@RequiredArgsConstructor
+@Getter
+public final class StandardTransactionContexts implements TransactionContexts {
+    
+    private final Map<String, ShardingTransactionManagerEngine> engines;
+    
+    public StandardTransactionContexts() {
+        this(new HashMap<>());
+    }
+    
+    @Override
+    public ShardingTransactionManagerEngine getDefaultEngine() {
+        return engines.get(DefaultSchema.LOGIC_NAME);
+    }
+    
+    @Override
+    public void close() throws Exception {
+        for (ShardingTransactionManagerEngine each : engines.values()) {
```

Comment:
why not lambda ?

Reply:
lambda cannot work if the inline code declare checked exception